.github/workflows/build_wasm.yml

```diff
@@ -5,7 +5,7 @@ on:
     inputs:
       ref:
         description: The ref to be used for the repo
-        default: dev
+        default: ''
```

Comment:
What does it mean by setting the default as ' '?

Reply:
It means when not provided, it will stay empty (instead of dev).
Later the actual checkout will then be decided by `github.ref` (running branch)
```
ref: ${{ github.event.inputs.ref || github.ref }}
```